portfolio/src/main/webapp/script.js

```diff
@@ -12,20 +12,54 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// The "fact number" of the previous fact given. "-1" is special value.
+let prevFactNum = -1;
+
 /**
  * Adds a random fun fact about myself to the page.
  */
-function addRandomFact() {
+function addFunFact() {
   const facts =
-      ['I have two older brothers!',
-       'I\'ve played tennis for fourteen years!', 
-       'My favorite book is Enlightenment Now by Steven Pinker.', 
-       'My favorite TV show is It\'s Always Sunny in Philadelphia.'];
+    ['I have two older brothers!',
+      'I\'ve played tennis for fourteen years!', 
+      'My favorite book is Enlightenment Now by Steven Pinker.', 
+      'My favorite TV show is It\'s Always Sunny in Philadelphia.'];
+
+  // Pick a random number corresponding to a fun fact.
+  let factNum = Math.floor(Math.random() * facts.length);
+
+  // If the same fact number is chosen twice consecutively, change the fact number.
+  if (factNum === prevFactNum) {
+    factNum++;
+    if (factNum == 4) {
+      factNum = 0;
+    }
```

Comment:
No action needed. Just as an aside, check out the modulus operator (ie. %) for an interesting alternate way to handle the case where factNum exceeds the bounds of the array.

Reply:
Ah, that's a good idea -- I didn't think of that. Thanks!